modules/gdscript/gdscript.cpp

```diff
@@ -2146,13 +2146,22 @@ void GDScriptLanguage::profiling_start() {
 		elem->self()->profile.last_frame_call_count = 0;
 		elem->self()->profile.last_frame_self_time = 0;
 		elem->self()->profile.last_frame_total_time = 0;
+		elem->self()->profile.native_calls.clear();
+		elem->self()->profile.last_native_calls.clear();
 		elem = elem->next();
 	}
 
 	profiling = true;
 #endif
 }
 
+void GDScriptLanguage::profiling_set_save_native_calls(bool opt) {
+#ifdef DEBUG_ENABLED
+	MutexLock lock(this->mutex);
```

Comment:
```suggestion
	MutexLock lock(mutex);
```
Unless this name is somehow ambiguous